src/base/memory_monitor_sr.cpp

```diff
@@ -239,6 +241,38 @@ bool mmon_is_memory_monitor_enabled ()
   return (mmon_Gl != nullptr);
 }
 
+int mmon_initialize (const char *server_name)
+{
+  int error = NO_ERROR;
+
+  assert (mmon_Gl == nullptr);
+  assert_release (server_name != NULL);
```

Comment:
This comment was made because it was judged that the code was written to mean that mmon initialization is allowed even if server_name is nullptr in release mode. Therefore, it would be better to leave `assert (server_name != nullptr)`.